lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -506,6 +506,21 @@
             - "Memory balloon is a guest device, which may be used to re-distribute / reclaim the host memory
                based on VM needs in a dynamic way. In this way it's possible to create memory over commitment states."
         version_added: "2.5"
+    numa_tune_mode:
+         description:
+            - "Set how the memory allocation for NUMA nodes of this VM is applied (relevant if NUMA nodes are set for this VM)."
+            - "It can be one of following: I(interleave), I(preferred) or I(strict)."
+            - "If no value is passed, default value is set by oVirt/RHV engine."
+        version_added: "2.5"
+    numa_node:
```

Comment:
should this be a list, so `numa_nodes`

Reply:
it's not a list. You add only one numa node each time as implemented in the api. See how __attach_numa_node() is implemented. Is it a problem?

Reply:
So how do you remove any NUMA node? With current implementation you can only add it , right?

Reply:
Right. It is a problem. How do you usually remove other entities that use a different service as part of ovirt_vms module? Should we add another dictionary value for marking the "delete" operation?